src/autoload.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * File required since elasticsearch-php 7.4.1 for aliasing
+ * to the previous endpoint class names.
+ *
+ * @see https://github.com/elastic/elasticsearch-php/issues/967
+ *
+ * NOTE: This file will be removed with elasticsearch-php 8.0.0
+ */
+
+class_alias('\Elasticsearch\Endpoints\Nodes\Info', '\Elasticsearch\Endpoints\Cluster\Nodes\Info');
+class_alias('\Elasticsearch\Endpoints\Nodes\Stats', '\Elasticsearch\Endpoints\Cluster\Nodes\Stats');
```

Comment:
What about previous classes:
- `Elasticsearch\Endpoints\Cluster\Nodes\HotThreads`
- `Elasticsearch\Endpoints\Cluster\Nodes\Usage`
- `Elasticsearch\Endpoints\Cluster\Nodes\ReloadSecureSettings`

Reply:
You right, just added it. Thanks for this code review!